src/components/TodoLine/TodoLine.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
+import { currentTodoSlice } from '../../features/currentTodo';
+import { useAppDispatch } from '../../app/hooks';
+
+type Props = {
+  todo: Todo;
+  isSelected: boolean;
+};
+
+export const TodoLine: React.FC<Props> = ({ todo, isSelected }) => {
+  const selectedTodoDispatch = useAppDispatch();
+
+  const changeCurrentTodo = (newTodos: Todo) => {
```

Comment:
The parameter `newTodos` in the `changeCurrentTodo` function should be renamed to `newTodo` to accurately reflect that it represents a single Todo object, not multiple Todos.